src/blocks/map/style.scss

```diff
@@ -106,3 +134,131 @@
 		margin-bottom: 0;
 	}
 }
+
+// These general non-layout styles are sandboxed to the side-by-side mode
+// as of July 2025. They could be expanded to apply to all maps if it is
+// acceptable to retroactively change design on e.g. old annual reports.
```

Comment:
We've updated a lot of the other styles on older annual reports. The line has been colors or styles specific to that report stay, anything like fonts, spacing, site width that should be site-wide get updated. I'm not sure where these fall, but it seems like changes would be ok, as long as everything still works correctly, and has sufficient color contrast, etc.

Reply:
Agree, but think we can maybe tackle that next time we work on the annual report. It was getting quite finicky right now :/